.circleci/config.yml

```diff
@@ -11,10 +11,13 @@ jobs:
       - image: circleci/python:3.6.1
         environment:
           FLASK_CONFIG: testing
+          DATABASE_URL: postgresql://root@localhost/circle_test
+          HOOK_SECRET_KEY: SECRETS
```

Comment:
@karlcow I know this is probably not how this should be set up, but I don't have access to set an environment variable in CircleCI (assuming that's the best way).

Reply:
@laghee which part do you think is wrong?
the DATABASE_URL or HOOK_SECRET_KEY?

`HOOK_SECRET_KEY` doesn't seem relevant as the testing will probably not interact with the real site, so a dummy key should be fine.